api/package/get-owned-packages.js

```diff
@@ -0,0 +1,9 @@
+const S = require('fluent-schema')
+const { PackageMaintainerProps } = require('../../types/Package')
+const { Base200WithData } = require('../../common/reply')
+
+module.exports = {
+  response: {
+    200: Base200WithData('packages', S.array().items(PackageMaintainerProps))
```

Comment:
these package maintainer props look a little dubious to me. i'm thinking this endpoint will only be called by auth'd maintainers. it looks like what they'll get back is the public package props but also a list of internal mongo IDs (with rev percentages) that correspond to the maintainers. thinking that the backend should first translate those IDs into something more useful to the frontend and the schema should reflect that. email addresses / names ? idk. is the front end going to take the list of user IDs and ask the backend to deref them all ?

Reply:
this route i haven't even revisit yet, i'm going to revisit when i get to that part, this was just about moving it to a new file since the previous get.js was logic for get-owned-packages